src/modules/shared/components/MediaCard/MediaCard.tsx

```diff
@@ -16,17 +17,33 @@ const MediaCard: FC<MediaCardProps> = ({
 	description,
 	keywords,
 	preview,
+	detailLink,
 	publishedAt,
 	publishedBy,
 	title,
 	type,
 	view,
-	onBookmark = () => null,
-	onTitleClick,
```

Comment:
Deze wordt nog gebruikt in de `MediaCardList` component, dus daar nog aanpassen naar het gebruik van `detailLink`